stable/locust/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: locust
-version: "0.9.19"
+version: "0.1.20"
```

This would be an older version as 1 is less than 9 🙂

```suggestion
version: "0.19.20"
```

---

Thanks @max-rocket-internet; I missed that. I committed the suggestion.